### PR TITLE
Update MC Dockerfile

### PR DESCRIPTION
**Pull request by SConaway**

### CODE REVIEW

File: minecraft-server/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN apk add -U \
 
 RUN pip install mcstatus
 
-HEALTHCHECK CMD mcstatus localhost ping
+HEALTHCHECK CMD mcstatus localhost:$SERVER_PORT ping
```
Comment (itzg) [inline minecraft-server/Dockerfile]: Does Docker delay the variable substitution here until the actual invocation?

**Comment by SConaway** [comment]
I think so, but I'll try it later today.

**Comment by itzg** [comment]
I was curious and was going to test it out later also :) .

**Comment by SConaway** [comment]
I just built this and it seems to work. I'm making a image to control a server also and using yours as guidance. I have one question though: How does `mc-server-runner` work? There's no readme, but it seems like something useful.

**Pull request merged**